docs/source/examples/shapes_images_positions.rst

```diff
@@ -1,278 +0,0 @@
-Shapes, Images and Positions
-=================================
-.. manim:: ShowScreenResolution
-    :save_last_frame:
-
-    class ShowScreenResolution(Scene):
-        def construct(self):
-            pixel_height = config["pixel_height"]  #  1080 is default
-            pixel_width = config["pixel_width"]  # 1920 is default
-            frame_width = config["frame_width"]
-            frame_height = config["frame_height"]
-            self.add(Dot())
-            d1 = Line(frame_width * LEFT / 2, frame_width * RIGHT / 2).to_edge(DOWN)
-            self.add(d1)
-            self.add(Tex(str(pixel_width)).next_to(d1, UP))
-            d2 = Line(frame_height * UP / 2, frame_height * DOWN / 2).to_edge(LEFT)
-            self.add(d2)
-            self.add(Tex(str(pixel_height)).next_to(d2, RIGHT))
-
-
-.. manim:: GeometricShapes
-    :save_last_frame:
-
-    class GeometricShapes(Scene):
-        def construct(self):
-            d = Dot()
-            c = Circle()
-            s = Square()
-            t = Triangle()
-            d.next_to(c, RIGHT)
-            s.next_to(c, LEFT)
-            t.next_to(c, DOWN)
-            self.add(d, c, s, t)
-
-.. manim:: PointMovingOnShapes
-
-    class PointMovingOnShapes(Scene):
-        def construct(self):
-            circle = Circle(radius=1, color=BLUE)
-            dot = Dot()
-            dot2 = dot.copy().shift(RIGHT)
-            self.add(dot)
-
-            line = Line([3, 0, 0], [5, 0, 0])
-            self.add(line)
-
-            self.play(GrowFromCenter(circle))
-            self.play(Transform(dot, dot2))
-            self.play(MoveAlongPath(dot, circle), run_time=2, rate_func=linear)
-            self.play(Rotating(dot, about_point=[2, 0, 0]), run_time=1.5)
-            self.wait()
-
-
-.. manim:: ManimCELogo
-    :save_last_frame:
-
-    class ManimCELogo(Scene):
-        def construct(self):
-            logo_green = "#87c2a5"
-            logo_blue = "#525893"
-            logo_red = "#e07a5f"
-            ds_m = MathTex(r"\mathbb{M}", z_index=20).scale(7)
-            ds_m.shift(2.25*LEFT + 1.5*UP)
-            circle = Circle(color=logo_green,
-                            fill_opacity=1,
-                            z_index=7)
-            square = Square(color=logo_blue,
-                            fill_opacity=1,
-                            z_index=5)
-            triangle = Triangle(color=logo_red,
-                                fill_opacity=1,
-                                z_index=3)
-            circle.shift(LEFT)
-            square.shift(UP)
-            triangle.shift(RIGHT)
-            self.add(triangle, square, circle, ds_m) # Order matters
-            self.wait()
-
-Download the resource for the following example `here <https://github.com/ManimCommunity/manim/blob/master/docs/source/_static/manim-logo-sidebar.svg>`_ 
```

Comment:
docs source

Reply:
@kolibril13 this was merged back in November - and I don't understand your comment here. Care to explain here?

Reply:
I needed to reference this line in pr #926 , and making a comment on this line was the fastest way to do so ;)